packages/base/src/asset-registries/Illustrations.ts

```diff
@@ -36,7 +36,11 @@ const registerIllustrationLoader = (illustrationName: string, loader: Illustrati
 const _loadIllustrationOnce = async (illustrationName: string) => {
 	if (!illustrationPromises.has(illustrationName)) {
 		if (!loaders.has(illustrationName)) {
-			throw new Error(`No loader registered for the ${illustrationName} illustration. Probably you forgot to import the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module.`);
+			if (illustrationName.startsWith("Tnt")) {
+				throw new Error(`No loader registered for the ${illustrationName} illustration. Probably you forgot to import the "@ui5/webcomponents-fiori/dist/illustrations/tnt/${illustrationName.replace(/^Tnt/, "")}.js" module. Or you can import all illustrations by importing the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module.`);
+			} else {
+				throw new Error(`No loader registered for the ${illustrationName} illustration. Probably you forgot to import the "@ui5/webcomponents-fiori/dist/illustrations/${illustrationName}.js" module. Or you can import all illustrations by importing the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module.`);
+			}
```

Comment:
The second part of the message

"Or you can import all illustrations by importing the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module."

we can update to explicitly say that only the used illustration will be fetched:

"Or you can import  the "@ui5/webcomponents-fiori/dist/illustrations/AllIllustrations.js" module that will make all illustrations available, but fetch only the ones used."

otherwise people may understand that all will be bundled, causing large bundle size